tools/dockerise/__init__.py

```diff
@@ -1,11 +1,17 @@
 #!/usr/bin/env python3
 
+import array
```

Comment:
Still have all these uneeded imports that are added, this file should not change